src/components/description-box/description-box.js

```diff
@@ -0,0 +1,74 @@
+import React, { PropTypes, Component } from 'react';
+import { canEditDescription } from '~/src/state/user/privileges';
+import marked from 'marked';
+// import './description-box.scss';
+
+export default class DescriptionBox extends Component {
+
+    static propTypes = {
+        activeItem: PropTypes.object.isRequired,
```

Comment:
Is there a standardized way to order these?  EG group by action/property then sort each group alphabetically.